### ISSUE TITLE
CloudvolumeGraphene incorrectly using gs:// paths and google authentication

### ISSUE THREAD

**Issue by ceesem** [seung-lab/cloud-volume#407]
With the graphene cloud path 'graphene://https://minniev1.microns-daf.com/segmentation/table/minnie3_v1' I am still running into gs:// path lookups. These do not occur with 2.0.1, but seem to start afterward.

```python
cv = cloudvolume.CloudVolume('graphene://https://minniev1.microns-daf.com/segmentation/table/minnie3_v1', use_https=True)
```

I'm getting the (slightly truncated) error:

```
Using default Google credentials. There is no ~/.cloudvolume/secrets/google-secret.json set.
```
```python
Empty                                     Traceback (most recent call last)
~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudfiles/connectionpools.py in get_connection(self, secrets, endpoint)
     53       try:
---> 54         conn = self.pool.get(block=False)
     55         self.pool.task_done()

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/queue.py in get(self, block, timeout)
    166                 if not self._qsize():
--> 167                     raise Empty
    168             elif timeout is None:

Empty: 

During handling of the above exception, another exception occurred:

DefaultCredentialsError                   Traceback (most recent call last)
<ipython-input-11-b9f39dafd7b3> in <module>
----> 1 cv = cloudvolume.CloudVolume('graphene://https://minniev1.microns-daf.com/segmentation/table/minnie3_v1' class="ansi-blue-fg">, use_https=True)

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudvolume/cloudvolume.py in __new__(cls, cloudpath, mip, bounded, autocrop, fill_missing, cache, compress_cache, cdn_cache, progress, info, provenance, compress, compress_level, non_aligned_writes, parallel, delete_black_uploads, background_color, green_threads, use_https, max_redirects, mesh_dir, skel_dir, agglomerate)
    202     path = strict_extract(cloudpath)
    203     if path.format in REGISTERED_PLUGINS:
--> 204       return REGISTERED_PLUGINS[path.format](**kwargs)
    205     else:
    206       raise UnsupportedFormatError(

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudvolume/datasource/graphene/__init__.py in create_graphene(cloudpath, mip, bounded, autocrop, fill_missing, cache, compress_cache, cdn_cache, progress, info, provenance, compress, parallel, delete_black_uploads, background_color, green_threads, use_https, mesh_dir, skel_dir, agglomerate, **kwargs)
     67     )
     68 
---> 69     mesh = GrapheneMeshSource(meta, cache, config)
     70     skeleton = PrecomputedSkeletonSource(meta, cache, config)
     71 

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudvolume/datasource/graphene/mesh/__init__.py in __new__(cls, meta, cache, config, readonly)
     10 class GrapheneMeshSource(object):
     11   def __new__(cls, meta, cache, config, readonly=False):
---> 12     mesh_meta = GrapheneMeshMetadata(meta, cache)
     13 
     14     if mesh_meta.is_sharded():

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudvolume/datasource/precomputed/mesh/metadata.py in __init__(self, meta, cache, info)
     15       self.info = info
     16     else:
---> 17       self.info = self.fetch_info()
     18 
     19   @property

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudvolume/datasource/precomputed/mesh/metadata.py in fetch_info(self)
     69       return self.default_info()
     70 
---> 71     info = self.cache.download_json(self.meta.join(self.mesh_path, 'info'))
     72     if not info:
     73       return self.default_info()

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudvolume/cacheservice.py in download_json(self, path, compress)
    352     Returns: content or None
    353     """
--> 354     res = self.download( [ path ], compress=compress, progress=False )
    355     res = res[path]
    356     if res is None:

~/opt/miniconda3/envs/analysis_dev/lib/python3.7/site-packages/cloudvolume/cacheservice.py in download(self, paths, compress, progress)
    475     for frag in remote_fragments:
    476       if frag['error'] is not None:
--> 477         raise frag['error']
    478 
    479     if self.enabled:
```

**Comment by ceesem** [comment]
In particular, the 'using default Google credentials` shouldn't occur if it's going through https:// requests with tokens.

**Comment by william-silversmith** [comment]
It looks like we fixed it for regular precomputed but not graphene. Let me add that in.

**Comment by william-silversmith** [comment]
It looks like when I added use_https to the Precomputed metadata parent class, it started clobbering the value in the Graphene metadata class. I fixed that in the update cited above.

**Comment by william-silversmith** [comment]
Fixed in Version 2.2.1
